packages/stripe-integration/src/stripe-upe/stripe-ocs-payment-strategy.ts

```diff
@@ -0,0 +1,413 @@
+                wallets: {
+                    applePay: StripeStringConstants.NEVER,
+                    googlePay: StripeStringConstants.NEVER,
+                },
+                layout: {
+                    type: 'accordion',
+                    defaultCollapsed: false,
+                    radios: true,
+                    spacedAccordionItems: false,
+                    visibleAccordionItemsCount: 0,
+                },
+            });
+
+        this.stripeUPEIntegrationService.mountElement(stripeElement, containerId);
+
+        stripeElement.on('ready', () => {
+            render();
+        });
+
+        stripeElement.on('change', (event: StripeEventType) => {
+            this._onStripeElementChange(event, gatewayId, methodId, paymentMethodSelect);
+        });
+
+        handleClosePaymentMethod?.(this._collapseStripeElement.bind(this));
+    }
+
+    private async _loadStripeJs(
+        stripePublishableKey: string,
+        stripeConnectedAccount: string,
+    ): Promise<StripeUPEClient> {
+        if (this.stripeUPEClient) {
+            return this.stripeUPEClient;
+        }
+
+        return this.scriptLoader.getStripeClient(stripePublishableKey, stripeConnectedAccount);
+    }
+
+    private _getElementAppearance(
+        style?: StripeUPEPaymentInitializeOptions['style'],
+    ): StripeUPEAppearanceOptions | undefined {
+        if (!style) {
+            return;
+        }
+
+        const titleFontSize = '15px'; // TODO: get style from theme
+        const titleFontWeight = '700'; // TODO: get style from theme
+        const titleColor = '#5f5f5f'; // TODO: get style from theme
+        const radioColor = '#ddd'; // TODO: get style from theme
+        const radioFocusColor = '#4496f6'; // TODO: get style from theme
+
+        return {
+            variables: {
+                ...this.stripeUPEIntegrationService.mapAppearanceVariables(style),
+                fontFamily: 'Montserrat, Arial, Helvetica', // TODO: get style from theme
+            },
+            rules: {
+                '.Input': this.stripeUPEIntegrationService.mapInputAppearanceRules(style),
+                '.AccordionItem': {
+                    borderRadius: 0,
+                    borderWidth: 0,
+                    borderBottomWidth: '1px',
+                    boxShadow: 'none',
+                    fontSize: titleFontSize,
+                    fontWeight: titleFontWeight,
+                    padding: '13px 20px 13px 18px',
+                },
+                '.TabLabel, .AccordionItem': {
+                    fontSize: titleFontSize,
+                    fontWeight: titleFontWeight,
+                    color: titleColor,
+                },
+                '.TabLabel--selected, .AccordionItem--selected': {
+                    fontWeight: titleFontWeight,
+                    color: titleColor,
+                },
+                '.RadioIcon': {
+                    // 26px / 0.88 = 29.54
+                    width: '29.54px',
```

Comment:
As far as I remember, floating numbers in pixels make rounds to closest full number, which in your case might be 30px 🤔

Reply:
Fully agree with you. And have a discussion about this values and how stripe calculate them.
But this values was provided by Stripe team and they sure that it will be good to send such values to the Stripe Elements Appearance API.
So if they expect to see this floating numbers and don't want to change it somehow - we decide to use this values and calculations.

P.S.: for now it is a hardcoded mock data and in future tasks this values will be calculated, to make it more transparent for our merchants.